Podcast/EpisodeCard.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  EpisodeCard.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 2/22/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class EpisodeCard: Card {
+    
+    var episodeID: Int
+    var episodeTitle: String = ""
+    var dateCreated: Date!
+    var time: Double!
+    var descriptionText: String!
+    var smallArtworkImageURL: URL?
+    var smallArtworkImage: UIImage!
+    var nRecommended: Int //number total out of user base that has recommended episode
+    var tags: [String]
+    var seriesTitle: String!
+    var isBookmarked: Bool!
+    var isRecommended: Bool!
+    
+    init(episodeID: Int, episodeTitle: String = "", dateCreated: Date = Date(), descriptionText: String = "Not avaliable", smallArtworkImage: UIImage = #imageLiteral(resourceName: "filler_image"), time: Double = 0.0, nRecommended: Int = 0, tags: [String] = [], seriesTitle: String = "", isBookmarked: Bool = false, isRecommended: Bool = false) {
+        self.episodeID = episodeID
```

Comment:
Agreed I added a commented out one that we can add in when we know what the json response looks like